app/helpers/accordion_helper.rb

```diff
@@ -1,8 +1,9 @@
 module AccordionHelper
-  def accordion(target_id, header_text, &block)
+  def accordion(target_id, header_text, options = {}, &block)
```

Comment:
I know people don't love comments here, but I think it might be helpful to enumerate what options are accepted, so that one doesn't have to look at (potentially) multiple files to figure out all the things they can do with the accordion.

Thoughts?